build.gradle

```diff
@@ -2,14 +2,18 @@ buildscript {
   ext.versions = [
       minSdk              : 21,
       compileSdk          : 28,
-      kotlin              : '1.2.71',
-      supportLib          : '28.0.0',
-      supportTest         : '1.0.2',
+      kotlin              : '1.3.11',
+      supportLib          : '1.0.0',
+      recyclerView        : '1.1.0-alpha01',
+      material            : '1.1.0-alpha02',
+      cardview            : '1.0.0',
+      room                : '2.1.0-alpha03',
+      supportTest         : '1.1.0-alpha4',
```

Comment:
@rakshakhegde can you check if there are stable versions for these dependencies? I'm a bit worried of using alpha.